.github/workflows/tests.yaml

```diff
@@ -6,3 +6,10 @@ jobs:
     with:
       context: tests/docker
       name: test
+
+  trivy:
+    needs: docker-build
+    name: Trivy Vulnerability Scan for Docker images
+    uses: scality/workflows/.github/workflows/trivy.yaml@main
```

Comment:
In this case, as you are inside the workflows repository, and you want to test your code, you should use the `./.` syntax.

But users outside of this repo will use the syntax you have now.

```suggestion
    uses: ./.github/workflows/trivy.yaml
```